tests/tck/job/Job.feature

```diff
@@ -98,6 +120,66 @@ Feature: Submit job space requirements
       """
     Then an ExecutionError should be raised at runtime: Save job failure!
 
+  Scenario: Submit and show jobs in other space
+    Given create a space with following options:
+      | partition_num  | 9                |
+      | replica_factor | 1                |
+      | vid_type       | FIXED_STRING(20) |
+    And wait 6 seconds
+    When executing query:
+      """
+      SUBMIT JOB COMPACT;
+      """
+    Then the result should be, in any order:
+      | New Job Id |
+      | /\d+/      |
+    And wait 1 seconds
+    When executing query:
+      """
+      SUBMIT JOB FLUSH;
+      """
+    Then the result should be, in any order:
+      | New Job Id |
+      | /\d+/      |
+    And wait 1 seconds
+    When executing query:
+      """
+      SUBMIT JOB STATS;
+      """
+    Then the result should be, in any order:
+      | New Job Id |
+      | /\d+/      |
+    And wait 10 seconds
+    When executing query:
+      """
+      SHOW JOBS;
+      """
+    Then the result should be, the first 3 records in order, and register Job Id as a list named job_id:
+      | Job Id | Command   | Status     | Start Time | Stop Time |
+      | /\d+/  | "STATS"   | "FINISHED" | /\w+/      | /\w+/     |
+      | /\d+/  | "FLUSH"   | "FINISHED" | /\w+/      | /\w+/     |
+      | /\d+/  | "COMPACT" | "FINISHED" | /\w+/      | /\w+/     |
+    Given create a space with following options:
+      | partition_num  | 9                |
+      | replica_factor | 1                |
+      | vid_type       | FIXED_STRING(20) |
+    When executing query:
+      """
+      SHOW JOBS;
+      """
+    Then the result should be, in order:
+      |Job Id|Command|Status|Start Time|Stop Time|
+    When executing query, fill replace holders with element index of 0 in job_id:
+      """
+      SHOW JOB {};
+      """
+    Then an ExecutionError should be raised at runtime:Job not in chosen space!
+    When executing query, fill replace holders with element index of 0 in job_id:
+      """
+      STOP JOB {};
+      """
+    Then an ExecutionError should be raised at runtime:Job not in chosen space!
```

Comment:
For these tests, you may need to make some enhancements to the TCK test framework

Reply:
Do you mean that I should give a sentence to match operation like "USE SPACE" instead of use current sentences like "create a space with following options"?

Reply:
This test case is enough I think